etc/adminhtml/system.xml

```diff
@@ -73,7 +73,25 @@
                         <field id="debug">1</field>
                     </depends>
                 </field>
-                <field id="connected" translate="label" sortOrder="12" type="hidden" showInDefault="1" showInWebsite="1" showInStore="1">
+                <field id="sandbox" translate="label" sortOrder="12" type="select" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Sandbox</label>
+                    <comment>Change all API requests to be made to TaxJar's sandbox account.</comment>
```

Comment:
@ryanreeves-taxjar Can we update the description here to include some more detail similar to the new notification?

```
Use the TaxJar sandbox environment to test API sales tax calculations. We recommend only using this mode in a staging environment. Some features may not work as expected.
```